crates/sui-core/src/quorum_driver/metrics.rs

```diff
@@ -72,37 +68,14 @@ impl QuorumDriverMetrics {
                 "Total attempt times of ok response",
                 mysten_metrics::COUNT_BUCKETS.to_vec(),
                 registry,
-            ).unwrap(),
+            )
+            .unwrap(),
             current_requests_in_flight: register_int_gauge_with_registry!(
                 "current_requests_in_flight",
                 "Current number of requests being processed in QuorumDriver",
                 registry,
             )
             .unwrap(),
-            total_err_process_tx_responses_with_nonzero_conflicting_transactions: register_int_counter_with_registry!(
```

Comment:
From the doc, this seems indicating the number of tx experiencing conflicting transaction, which is still useful metrics to have. Is `total_client_double_spend_attempts_detected` the same thing as `the number of tx that experiencing conflicting tx, and not able to execute`?

Reply:
`total_err_process_tx_responses_with_nonzero_conflicting_transactions` was actually probably poorly named originally as it was meant to track the retryable tx with non zero conflicting tx. The metric you pointed to, `total_client_double_spend_attempts_detected`, will track all of the non retryable tx that had non zero conflicting tx. So we will be able to see the number of tx that fail due to experiencing conflicting transactions.